crypto/cryptlib.c

```diff
@@ -326,7 +358,7 @@ void OPENSSL_die(const char *message, const char *file, int line)
     /*
      * Win32 abort() customarily shows a dialog, but we just did that...
      */
-# if !defined(_WIN32_WCE)
+# if !defined(_WIN32_WCE) && !defined(OPENSSL_SYS_WIN_CORE)
     raise(SIGABRT);
```

Comment:
There is raise in api-ms-win-crt-runtime, so why not? Point with raise is that it's supposed to generate core dump.

Reply:
good point, fixed